### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by naanmohammed**
In this milestone of the vet clinic project :

 - Species column was added
 - New entries of data were added
 - New and more complex queries were used to analyze the data.

The screenshots required are attached below. Thank you and looking forward to your review.

Screenshot (308)

Screenshot (309)

Screenshot (310)

Screenshot (311)

Screenshot (312)
7)

Screenshot (313)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,38 @@ SELECT date_of_birth FROM animals WHERE name IN ('Augmon', 'Pikachu');
...
+ROLLBACK TO SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts > 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, COUNT(*) AS escape_attempts FROM animals GROUP BY neutered ORDER BY escape_attempts DESC;
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31' GROUP BY species;
```
Comment (alexander16108) [inline queries.sql]: the result of the following query are incorrect please fix them and try again

- How many animals have never tried to escape?
- Who escapes the most, neutered or not neutered animals?

the result of the first is 2 and the second is 7 t to 3 f

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @naanmohammed , 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- please fix the broken link in your mit license readme file it is highly unprofessional to leave broken links in your readme file 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by naanmohammed** [comment]
Updated according to requirements 
image

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @naanmohammed,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**